typescript/infra/config/environments/mainnet3/warp/getAncient8EthereumUSDCWrapConfig.ts

```diff
@@ -0,0 +1,49 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainMap,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+  buildAggregationIsmConfigs,
+  defaultMultisigConfigs,
+} from '@hyperlane-xyz/sdk';
+
+import { tokens } from '../../../../src/config/warp.js';
+
+export const getAncient8EthereumUSDCWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  const ismConfig = buildAggregationIsmConfigs(
+    'ethereum',
+    ['ancient8'],
+    defaultMultisigConfigs,
+  ).ancient8;
+
+  const ethereum: TokenRouterConfig = {
+    ...routerConfig.ethereum,
+    type: TokenType.collateral,
+    token: tokens.ethereum.USDC,
+    interchainSecurityModule: ismConfig,
+    // This hook was recovered from running the deploy script
+    // for the hook module. The hook configuration is the Ethereum
+    // default hook for the Ancient8 remote (no routing).
+    hook: '0x19b2cF952b70b217c90FC408714Fbc1acD29A6A8',
```

Comment:
TBD on a better way to set this

Reply:
I wish I documented why this was needed 😭
For now I think this is fine to keep as-is though